### ISSUE TITLE
No refile option presented when creating node

### ISSUE THREAD

**Issue by danielcontro** [chipsenkbeil/org-roam.nvim#61]
I use the LazyVim distribution and I installed the plugin using the configuration suggested in the README.
When creating a new node upon save the prompt for refiling the node to a new file isn't displayed.

**Comment by chipsenkbeil** [comment]
Can you elaborate a bit more? You're using 0.1.0 of this plugin and 0.3.4 of orgmode?

How are you creating the node? With org roam's find? Org roam's capture? Org roam's insert?

And how are you trying to save the node?  From the capture buffer, saving and quitting the buffer should trigger it. 

What version of neovim are you using?

**Comment by danielcontro** [comment]
I'm on Neovim 0.10.1, using version 0.1.0 of the plugin and 0.3.4 of orgmode; the following is my plugin's configuration:
  
``` Lua
{
  "chipsenkbeil/org-roam.nvim",
  tag = "0.1.0",
  dependencies = {
    {
      "nvim-orgmode/orgmode",
      tag = "0.3.4",
    },
  },
  event = "VeryLazy",
  config = function()
    require("org-roam").setup({
      directory = "~/Org/notes",
      org_files = {
        "~/Org/agenda/*",
        "~/Org/refile.org",
      },
    })
  end,
}
```

I'm creating the node with the `<Leader>nf` shortcut for org roam's find, saving it with `:w` and quitting the buffer with `:q`; I'll also attach a screencast of the behaviour of the plugin.


https://github.com/user-attachments/assets/9caeb429-691e-4e1b-9743-f7eca5579c27

**Comment by chipsenkbeil** [comment]
@danielcontro what plugin are you using for input that makes the floating buffer? It actually looks like it might be trying to ask you to confirm the refile, and then exits.

Anyway, capturing and refiling is leveraging the main orgmode plugin, so this is probably an issue with it. What happens if you disable the custom input plugin you're using?

**Comment by chipsenkbeil** [comment]
cc @kristijanhusak in case he's seen this capture issue before.

**Comment by danielcontro** [comment]
The custom input plugin is [noice.nvim](https://github.com/folke/noice.nvim), I tried disabling it and the creation of new nodes worked correctly presenting the refiling option thus I guess the issue isn't related to org-roam itself.

Though now I wonder why the selection of the capture template worked correctly while instead the input for refiling didn't as they both are input requests, perhaps orgmode handles it differently?

**Comment by chipsenkbeil** [comment]
@danielcontro yeah, there may be something with how we leverage orgmode's API that causes an issue if traditional refiling with the orgmode plugin works with `noice.nvim` and not with this plugin. But I don't know what that would be :/

Let's see if @kristijanhusak has any thoughts to diagnose/debug.

**Comment by danielcontro** [comment]
I slightly changed the plugin configuration as follows:
```Lua
{
    "chipsenkbeil/org-roam.nvim",
    tag = "0.1.0",
    dependencies = {
      {
        "nvim-orgmode/orgmode",
        tag = "0.3.4",
        event = "VeryLazy",
        ft = { "org" },
        config = function()
          require("orgmode").setup({
            org_agenda_files = "~/Org/agenda/*",
            org_default_notes_file = "~/Org/refile.org",
          })
        end,
      },
    },
    event = "VeryLazy",
    ft = { "org" },
    config = function()
      require("org-roam").setup({
        directory = "~/Org/notes",
        org_files = {
          "~/Org/agenda/*",
          "~/Org/refile.org",
        },
      })
    end,
  }
```

I tested the default orgmode capture and it has no issues. It should be noted though that the default functionality doesn't request a confirmation but simply inserts the capture in the specified refiling file.

**Comment by kristijanhusak** [comment]
> The custom input plugin is [noice.nvim](https://github.com/folke/noice.nvim), I tried disabling it and the creation of new nodes worked correctly presenting the refiling option thus I guess the issue isn't related to org-roam itself.

Do you also get the confirmation when testing with disabled `noice`? If yes, what does it say?

> I tested the default orgmode capture and it has no issues. It should be noted though that the default functionality doesn't request a confirmation but simply inserts the capture in the specified refiling file.

It does ask for a confirmation in case the buffer is modified, and you try to force quit with `:q!`. I don't know if with org-roam there are some modifications in the refiling process that triggers that confirmation, so answer to the first question will be helpful.

Since the issue seems to happen only with [noice.nvim](https://github.com/folke/noice.nvim), which as it states in the description is "highly experimental", I would say there isn't anything to fix here or in orgmode. Once we conclude, I would suggest to open up an issue in `noice` and see if there's a way to fix it.

**Comment by seflue** [comment]
I am pretty sure, that this issue is related with my fix addressed by PR #64. @danielcontro Can you checkout my branch and check, if this resolves your problem? For me, refiling to newly created Roam nodes works with this fix.
